data/korath/korath.txt

```diff
@@ -919,20 +1042,28 @@ outfitter "Korath Exiles"
 
 outfitter "Korath Exiles Remnant Donations"
 	"Banisher Grav-Turret"
-	"Fire-Lance"
-	"Grab-Strike Turret"
+	"Expeller Grav-Ray"
+	"Digger Mining Turret"
+	"Digger Mining Beam"
 	"Warder Anti-Missile"
+	"Firelight Storage Rack"
+	"Firestorm Torpedo Rack"
 	"Steering (Asteroid Class)"
+	"Steering (Comet Class)"
 	"Steering (Lunar Class)"
 	"Steering (Planetary Class)"
 	"Thruster (Asteroid Class)"
 	"Thruster (Comet Class)"
+	"Thruster (Lunar Class)"
 	"Thruster (Planetary Class)"
+	"Reverser (Asteroid Class)"
+	"Reverser (Comet Class)"
+	"Reverser (Lunar Class)"
+	"Engine (Meteor Class)"
+	"Bow Drive (Meteor Class)"
+	"Afterburner (Asteroid Class)"
+	"Afterburner (Comet Class)"
+	"Generator (Candle Class)"
 	"Generator (Furnace Class)"
-	"Triple Plasma Core"
-	"Small Heat Shunt"
-	"Large Heat Shunt"
+	"Generator (Inferno Class)"
 	"Fuel Processor"
-	"Systems Core (Small)"
-	"Systems Core (Medium)"
-	"Systems Core (Large)"
```

Comment:
Aren't some of these sold by the Remnant?

Reply:
They are, but probably shouldn't be.  If the remnant are selling them, I think it's because they're stealing them without permission, if that makes any sense, and are able to sell them due to their ubiquitousness.

Reply:
The stuff the Remnant sell is somewhat problematic. The actual in-game lore is that all these outfits are things that the Remnant have looted or "looted" in sufficient quantity that they're basically available in the same quantities as any regularly purchasable good.

Originally, these lists were limited to those outfits found on the various Korath ships that raid the Remnant, but some of those were pretty good pieces of outfits, and people argued for balances purposes that the list should be altered to just have the smaller/lesser versions of those outfits instead (despite the fact that some of those lesser outfits aren't used on any of the Korath raiders...) I went with it due to a) balance, and b) it becomes something of a  teaser for future Korath gear acquisition.

In any case, there's still the fact that the Korath could well be filling their ships with old versions of that smaller/lesser stuff (and it makes sense that they'd do so) specifically for this "trade", and that, rather than looting directly from equipped outfits, is the actual source of Remnant sold Korath things.

Reply:
As to the changes in specific. The fire-lance and grab-strike should probably stay on the list, as well as the small heat shunt. Those are heavily used items that they'll definitely have large numbers of heavily worn inventory. Likewise for the small systems core. 

Removing the large heat shunt, medium and large systems core, and the triple plasma core are OK if you wish as being perhaps "too high in value" for such a trade.

Reply:
Most definitely, yes.

I'd imagine that, over the years, the selection of stuff varied dramatically, depending on a variety of factors. It's also a bit of a forward investment, too. Obviously neither side knows what the other is going to have available to loot; so it'll be based on what the previous side has brought in the past. If the Korath brought more interesting things last time, the Remnant will bring more resources this time. And vice versa, of course. So on both sides it's a fine balance of:
- What do we have available currently?
- How much do we need now?
- How much will we need next time?
- How far can we push our advantage this time?
- How confident are we that we will get good value (aka did they short us last time?)
- How concerned are we that what we bring will be used against us?
- What alternatives do we have?

For instance, value or no, if there was an accident on a key production ship and the Korath are going to need larger quantities of resources now and in the future, it may not matter if all they have available is the "good stuff". On the other hand, if they've found a huge swath of fairly pure minerals in asteroids, they might be able to cut back and just do volumes of simple stuff.
 
This loot list is thus kind of abstracted, and I suppose comes down to "How badly off are the Korath (just averaging out all the random life and everything else) are the Korath at the start of the game?